exercises/kindergarten-garden/canonical-data.json

```diff
@@ -0,0 +1,126 @@
+{
+  "exercise": "kindergarten-garden",
+  "version": "1.0.0",
+  "comments": [
+    "Garden can be created with just a diagram or a diagram and students.",
+    "The default students are:",
+    "  Alice, Bob, Charlie, David, Eve, Fred, Ginny, Harriet, Ileana, Joseph, Kincaid, and Larry."
+  ],
+  "cases": [
+    {
+      "description": "partial garden tests",
+      "cases": [
+        {
+          "description": "test Alice's garden",
+          "property": "plants",
+          "diagram": "RC\nGG",
+          "student": "Alice",
+          "expected": ["radishes", "clover", "grass", "grass"]
+        },
+        {
+          "description": "test different garden for Alice",
+          "property": "plants",
+          "diagram": "VC\nRC",
+          "student": "Alice",
+          "expected": ["violets", "clover", "radishes", "clover"]
+        },
+        {
+          "description": "test Bob's garden",
+          "property": "plants",
+          "diagram": "VVCG\nVVRC",
+          "student": "Bob",
+          "expected": ["clover", "grass", "radishes", "clover"]
+        },
+        {
+          "description": "test Bob and Charlie's garden",
+          "cases": [
+            {
+              "description": "test Bob's garden",
+              "diagram": "VVCCGG\nVVCCGG",
+              "property": "plants",
```

Comment:
anyone have a desire for consistent ordering? notice that property comes after diagram here, but before diagram in above cases